lib/opentelemetry/trace.ts

```diff
@@ -0,0 +1,3 @@
+import { NoopSpanProcessor } from "@opentelemetry/sdk-trace-base";
+
+export const noopSpanProcessor = new NoopSpanProcessor();
```

Comment:
We don't collect trace yet

We need to pass this processor to `registerOTel()` to avoid auto configuration of implicit processor to use Vercel Collector